charts/prefect-agent/values.yaml

```diff
@@ -1,62 +1,89 @@
-replicaCount: 1
+## Common parameters
+## nameOverride String to partially override common.names.name
+nameOverride: ""
+##  fullnameOverride String to fully override common.names.fullname
+fullnameOverride: ""
+## namespaceOverride String to fully override common.names.namespace
+namespaceOverride: ""
+## commonLabels Labels to add to all deployed objects
+commonLabels: {}
+## commonAnnotations Annotations to add to all deployed objects
+commonAnnotations: {}
 
-image:
-  repository: prefecthq/prefect
-  tag: 2-latest
-  pullPolicy: IfNotPresent
+agent:
+  image:
+    repository: prefecthq/prefect
+    tag: 2-latest
```

Comment:
should this use the .Chart.AppVersion by default? We can set this at chart packaging time using `helm package --app-version`, and users can still override it using `--set agent.image.tag` during `helm install`

Reply:
i don't think so, since it won't have any relation to our image versioning process